torchci/lib/bot/ciflowPushTrigger.ts

```diff
@@ -137,7 +137,6 @@ async function handleLabelEvent(context: Context<"pull_request.labeled">) {
   const valid_labels = [
     "ciflow/trunk",
     "ciflow/periodic",
-    "ciflow/all",
     "ciflow/android",
     "ciflow/binaries",
```

Comment:
Looks like "ciflow/binaries" are kept so we should be ok, I thin we need this label.